common/v2/components/AccountList.tsx

```diff
@@ -228,7 +253,12 @@ function buildAccountTable(
       const bodyContent = [
         <Label key={index}>
           <SIdenticon address={account.address} />
-          <Typography value={label} />
+          <LabelWithWallet>
+            <Typography value={label} />
+            <WalletTypeLabelContainer>
```

Comment:
This container div seems unnecessary, but we can address it later.

Reply:
We don't need a styled div, but we do seem to need the div still so that it doesn't stretch to fill the whole length.